public/app/features/datasources/settings/ButtonRow.tsx

```diff
@@ -23,7 +24,7 @@ const ButtonRow: FC<Props> = ({ isReadOnly, onDelete, onSubmit, onTest }) => {
       <button type="submit" className="btn btn-danger" disabled={isReadOnly} onClick={onDelete}>
         Delete
       </button>
-      <a className="btn btn-inverse" href="/datasources">
+      <a className="btn btn-inverse" href={datasourcesPageHref}>
```

Comment:
Think the simple solution is just to remove first forward slash. Then it will be relative to base href which is updated if you have appSubUrl

Reply:
Wouldn't it be relative to base href + the current directory you're on? We're using appSubUrl everywhere else, why not here?

Reply:
No it’s always relative to baseHref I think (when you use base href). Was not aware we use appSubUrl in hrefs j. Frontend thought only in backend

Reply:
I'm not a big fan of urls without leading slash @torkelo, especially that the behaviour of with/without leading slash isn't that obvious (i.e. see Marcus's comment).

In this case I have introduced the prop, to avoid config import in this dummy component. I think we should perform this navigation in callback prop (`onBackButtonClick`) if we had a location service or something alike.

Reply:
Yes, maybe it’s best. Can’t remember what the advantage was for using relative to base href was

Reply:
Maybe something related to how angular route/location service works?

Reply:
Think I might be have mixed it up with something else, I know we have had issues with using absolute urls (as that require an accurate root_url so limit that to only when it's needed).

So I think think this is fine, but do not think we need a prop for this, there is no reason not to use config directly in ButtonRow.

Reply:
Yes, you are right Torkel. I was thinking about making ButtonRow as dummy as possible, but since it's not grafana/ui component we can use app config there.